name 'MeTTaLog' is not defined
[()]
[(Error (assertEqual (llm (Agent <motto.agents.agent.EchoAgent object at 0x7f797c66f710>) (Script nested_script_main.mps)) "system Help the user with lunch order
user Hi there") 
Expected: ["system Help the user with lunch order
user Hi there"]
Got: [(Error (llm (Agent <motto.agents.agent.EchoAgent object at 0x7f797c66f710>) (Script nested_script_main.mps)) Exception caught:
NameError: name 'MeTTaLog' is not defined

At:
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/metta-motto/tests/motto/llm_gate.py(146): get_llm_args
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/metta-motto/tests/motto/llm_gate.py(193): llm
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/metta-motto/tests/motto/llm_gate.py(240): <lambda>
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/hyperon-experimental_python/hyperon/atoms.py(346): execute
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/hyperon-experimental_python/hyperon/atoms.py(176): _priv_call_execute_on_grounded_atom
  /opt/logicmoo_workspace/packs_sys/logicmoo_opencog/MeTTa/metta-wam/tests/python_compat/hyperon-experimental_python/hyperon/runner.py(31): run_step
  (17): run_step
)]
Missed result: "system Help the user with lunch order
user Hi there")]
0.43user 0.09system 0:00.53elapsed 99%CPU (0avgtext+0avgdata 128688maxresident)k
0inputs+8outputs (0major+28381minor)pagefaults 0swaps
